examples/client/Locomotion/src/pages/Payments/cashPaymentMethod.ts

```diff
@@ -0,0 +1,19 @@
+import { PaymentMethodInterface } from '../../context/payments/interface';
+
+const cashPaymentMethod : PaymentMethodInterface = {
+  brand: 'cash',
+  createdAt: new Date(),
+  customerId: '',
+  expiresAt: new Date(2100, 9, 9),
```

Comment:
It can be different interface because cash doesn't have all these values

Reply:
i want it to be generic for sending to payment ms the same object

Reply:
But you don't send the payment method to payment ms